com.github.Resticity.yml

```diff
@@ -0,0 +1,48 @@
+id: com.github.Resticity
+runtime: org.gnome.Platform
+runtime-version: '45'
+sdk: org.gnome.Sdk
+
+
+command: resticity
+finish-args:
+  - --share=ipc # Share IPC namespace with the host (necessary for X11).
+  - --socket=x11
+  - --device=dri # OpenGL rendering support.
+  - --talk-name=org.freedesktop.Notifications
+  - --filesystem=home
+  - --filesystem=host
+  - --socket=wayland
+  - --share=network
+  - --talk-name=org.kde.StatusNotifierWatcher
+
+build-options:
+  build-args:
+    - --share=network
+  env:
+    - GOROOT=/usr/lib/sdk/golang
```

Comment:
this is empty right now. you need to specify the use of the SDK extension.

My guess is that unlike you said you did, you didn't try to build this locally.